packages/vite/src/node/importGlob.ts

```diff
@@ -126,7 +133,18 @@ export async function transformImportGlob(
       ;[importee] = await normalizeUrl(file, pos)
     }
     imports.push(importee)
-    if (assertion?.assert?.type === 'raw') {
+    if (
+      options?.as === 'raw' ||
+      // TODO remove assert syntax for the Vite 3.0 release.
+      options?.assert?.type === 'raw'
+    ) {
+      logger.warn(
```

Comment:
Wont you want to log this only out if assert.type was used?

Reply:
Ah, sorry, yes. Commit pushed.